backend/src/test/resources/tests/query/SECONDARY_ID_MIXED/SECONDARY_IDS_MIXED.test.json

```diff
@@ -105,8 +107,12 @@
             "type": "REAL"
           },
           {
-            "name": "datum",
-            "type": "DATE_RANGE"
+            "name": "datum_start",
+            "type": "DATE"
+          },
+          {
+            "name": "datum_end",
+            "type": "DATE"
           },
```

Comment:
Musstest du hier von einer Column auf zwei wechseln?

Reply:
Oki, auf diese form sollten wir uns auch erstmal konzentieren, ich schätze, dass sie häufiger in Datensätzen vorkommt.